dist/scss/wds-components/_notifications.scss

```diff
@@ -47,15 +47,22 @@
 		max-width: $content-padding * 2 + 320px;
 		min-height: 80px;
 		min-width: $content-padding * 2 + 320px;
-		white-space: normal;
 		width: 100%;
+
+		// that's here because we need to override DS dropdowns list styles
```

Comment:
imagine it's 2018 - would you understand that comment?
why do we need to override it?